src/Navbar.css

```diff
@@ -0,0 +1,52 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 3em;
+  color: white;
+  background-color: #5757ff;
+}
+
+.col-1 {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+.btn {
+  background-color: transparent;
+  border: none;
+}
+
+.fa-chevron-down, .fa-plus {
+    margin-left: 0.5em;
+}
+
+.col-2 {
```

Comment:
If you have `.col-1` so no need to repeat all properties, you can just create a new class for padding.